.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Launch.json for VSCode for launching the app in debugging mode.  You can configure which script you want debugging mode to launch with the "command" value (currently set to yarn dev).  Breakpoints and call stack will be available in VSCode when running with this launch config